CHANGE_LOG.md

```diff
@@ -1,5 +1,7 @@
 # 0.3.0
 - Added overridable bobs and singles (`-b`/`--bob` for Bobs, `-n`/`--single` for Singles)
+- Special case `Grandsire` and `Stedman` on odd stages for the usual calls
+- Special case `Plain Hunt` on any stage 
```

Comment:
I think these could be a bit more descriptive - how about?:
```
- Make default calls correct for `Grandsire` and `Stedman` on any even stage
- Make Wheatley recognise `Plain Hunt` as a method on any stage
```

Reply:
Done